### PR TITLE
Allow specify user context in GitHub Status window

### PR DESCRIPTION
**Pull request by phracek**
Allow specify user context in GitHub Status window

    The context after this modification looks like:
    In case of context is not specified, then status is:
    `Testing Farm - <OS> - <version>`

    In case of context is specified, then status is:
    `Testing Farm - <context> <OS> - <version>`

### CODE REVIEW

File: export_variables.sh [hunk 1]
```diff
@@ -36,7 +37,10 @@
     ;;
 esac
 
-
+$context_prefix=""
```
[the remark this answers is not in the document]
Comment (jamacku) [reply export_variables.sh:40]: @phracek, you probably don't want the value of `context_prefix`. Just remove `$`

File: export_variables.sh [hunk 2]
```diff
@@ -36,7 +37,10 @@
...
+if [ "$user_context" != "undefined" ]; then
+  $context_prefix="$user_context - "
```
[the remark this answers is not in the document]
Comment (jamacku) [reply export_variables.sh:42]: The same here.

**Comment by phracek** [comment]
@jamacku Thanks for review. Can you please have a look at it once more?

File: export_variables.sh [hunk 3]
```diff
@@ -31,7 +35,10 @@ case "$test_case" in
     ;;
 esac
 
-
+context_prefix=""
+if [ "$user_context" != "undefined" ]; then
+  context_prefix="$user_context -"
```
Comment (jamacku) [inline export_variables.sh:40]: I would prefer this:

```sh
if [[ -n "$user_context" ]]; then
  context_prefix="$user_context - "
fi
```

And then you can drop the above if condition.

File: export_variables.sh [hunk 4]
```diff
@@ -6,12 +6,16 @@ all_os="$public_ranch $private_ranch"
 
 os_test="$1" # options: centos7, c8s, c9s, fedora, rhel7, rhel8, rhel9, rhel9-unsubscribed
 test_case="$2" # options: container, openshift-4
+user_context="$3"
```
Comment (jamacku) [inline export_variables.sh:9]: Maybe also add a comment.

File: action.yml [hunk 5]
```diff
@@ -38,6 +41,17 @@ runs:
           echo "path=$_path" >> "$GITHUB_OUTPUT"
         fi
 
+    - name: Set user defined context
+      id: user_context
+      shell: bash
+      run: |
+        ctx=${{ inputs.context }}
+        if [ "$ctx" == "" ]; then
+          echo "ctx=undefined >> "$GITHUB_OUTPUT"
```
Comment (jamacku) [inline action.yml:50]: I think you can pass `inputs.context` directly.

**Review by jamacku — CHANGES_REQUESTED** [review]

**Comment by zmiklank** [comment]
@phracek, could you please share why we need this feature? What will be specified in the <context>?

**Review by jamacku — APPROVED** [review]
LGTM

**Pull request merged**